host.go

```diff
@@ -74,8 +74,8 @@ func NewHost(cfg *Config) (*Host, error) {
 
 	key, err := loadKey(cfg.KeyFile)
 	if err != nil {
-		log.Debugf("failed to load libp2p key, generating key %s...", cfg.KeyFile)
```

Comment:
I disliked that this message used the word "fail", as I frequently search our logs for that word.